content/en/docs/components/pipelines/sdk/output-viewer.md

```diff
@@ -224,13 +228,14 @@ confusion matrix CSV file as a string in `source` field directly.
         {'name': 'predicted', 'type': 'CATEGORY'},
         {'name': 'count', 'type': 'NUMBER'},
       ],
-      'source': <CONFUSION_MATRIX_CSV_FILE>,
```

Comment:
I'd probably prefer the old format of leaving user to fill the source field, because most of the cases, we prefer putting the code that uploads confusion matrix csv file to a blob storage and generating the metadata to one single component.

Reply:
Got it, reverting the change to reference CSV file here.